src/assignment/hooks/assignment-teacher-tabs.tsx

```diff
@@ -34,10 +34,15 @@ export function useAssignmentTeacherTabs(
 					label: tText('assignment/hooks/assignment-tabs___details'),
 					icon: IconName.settings as IconName,
 				},
+				{
+					id: ASSIGNMENT_CREATE_UPDATE_TABS.PUBLISH,
+					label: tText('Publicatiedetails'),
```

Comment:
Shouldn't this be something like 'assignment/hooks/assignment-teachter-tabs__publication-details' ?